packages/backend/src/db/issue/issue.schema.ts

```diff
@@ -0,0 +1,36 @@
+import { Prop, Schema, SchemaFactory } from '@nestjs/mongoose';
+import { Document, Types } from 'mongoose';
+
+export type IssueDocument = Issue & Document;
+
+@Schema()
+export class Issue {
+  @Prop()
+  name: string;
+
+  @Prop()
+  description: string;
+
+  @Prop({ type: Types.ObjectId, ref: 'House' })
+  house: Types.ObjectId;
+
+  @Prop({ type: Types.ObjectId, ref: 'User' })
+  logger: Types.ObjectId;
+
+  @Prop()
+  loggedDate: Date;
```

Comment:
Everything looks great! I just had a minor question about this part. The date will be automatically created right? so does it need to have some default value or is it already done automatically?

Reply:
Yep, I just added a default value now 👍